app/subscriber/src/features/search-page/PreviousResults.tsx

```diff
@@ -29,54 +29,42 @@ export const PreviousResults: React.FC<IPreviousResultsProps> = ({
 
   const [prevResults, setPrevResults] = React.useState<IPreviousDate[]>([]);
 
-  React.useEffect(() => {
-    // wait for startDate, and also do not want to fetch if results are returned
-    if (!startDate || currDateResults.length) return;
-    createDateRanges(startDate);
-    // only want to run when start date or source ids change
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [filter.startDate, currDateResults, prevDateResults]);
-
   const createDateRanges = React.useCallback(
-    (startDate: Date) => {
+    (startDate: Moment) => {
       if (!prevDateResults.length) return setPrevResults([]);
-      const dayInMs = 24 * 60 * 60 * 1000; // Hours*Minutes*Seconds*Milliseconds
-      // Previous 7 days that will be used to fetch in a filter
-      let dateRanges: IPreviousDate[] = [];
-      const days =
-        moment(startDate)
-          .startOf('day')
-          .diff(
-            moment(prevDateResults[prevDateResults.length - 1].publishedOn).endOf('day'),
-            'days',
-          ) + 1;
-
-      // Generate the previous 7 days from the read date
-      for (let i = 1; i <= days; i++) {
-        const currStartDate = new Date(startDate.getTime() - i * dayInMs);
-        const currEndDate = new Date(currStartDate.getTime() + dayInMs - 1);
-        const currResults = prevDateResults.filter((res) => {
-          const resDate = new Date(res.publishedOn);
-          if (
-            resDate.getTime() >= currStartDate.getTime() &&
-            resDate.getTime() <= currEndDate.getTime()
-          ) {
-            return true;
-          }
-          return false;
+      const date = moment(startDate).startOf('day');
+      const dateRanges: IPreviousDate[] = [];
+      for (let i = 1; i < 9; i++) {
```

Comment:
Include an additional day because 24/48 hour filters are relative to the time of the search.